_posts/2022-07-11-Bazel-IntelliJ-Update.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: posts
+title: "Announcing Bazel & JetBrains co-maintenance of IntelliJ IDEA Bazel Plugin"
+authors:
+  - radvani13
+---
+
+### **Announcing Bazel & JetBrains co-maintenance of IntelliJ IDEA Bazel Plugin**
+
+Bazel plugin for IntelliJ IDEA announcement
+
+We are excited to announce that JetBrains* has joined as co-maintainer of the open source [Bazel plugin for IntelliJ IDEA](https://ij.bazel.build/). Using this plugin, developers can import Bazel project files and run Bazel tests and binaries within IntelliJ IDEA. JetBrains has already contributed to the 2022.1 and 2022.2 compatibility releases, so the community may be familiar with their work.
+
+  
+As co-maintainer, the JetBrains team reviews and merges contributions, collaborates on the roadmap, and monitors the CI pipeline, including the overall health of the [plugin’s Github repo](https://github.com/bazelbuild/intellij). This new maintenance structure enables faster review and merger of developer requests — such as support for Scala — and support for more contributions to and iteration of the plugin.
+
+ We welcome you to join the discussion! Discuss your feature request ideas and contributions by joining the [Bazel-Intellij plugin Slack channel](https://bazelbuild.slack.com/archives/C025SBYFC4E) or contacting the [Bazel Product team](mailto:product@bazel.build)[.](https://bazel.build/contribute)
```

Comment:
That doesn't seem like the right pairing.
Slack is impermanent, so we can't do design there, and product@bazel.build is too Google private.  Do we need new group for this?

Reply:
we can swap bazel intelliJ slack channel with a google group for IntelliJ. for product channel will replace with bazel-discuss